### PR TITLE
Bug/jenkins 42975 no hover text for edit and close buttons on toolbar

### PR DESCRIPTION
**Pull request by NicuPascu**
# Description
This PR adds a tool-tip to the close button, for the edit button: https://github.com/jenkinsci/blueocean-pipeline-editor-plugin/pull/46

See [JENKINS-42975](https://issues.jenkins-ci.org/browse/JENKINS-42975).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
@NicuPascu given this is green - could you attach a gif or something so @brody or @i386 can bless it? (should be only review needed I think)

**Comment by michaelneale** [comment]
@NicuPascu will need a merge-a-thon

File: blueocean-core-js/src/js/components/ResultPageHeader.jsx [hunk 1]
```diff
@@ -5,20 +5,25 @@ import {
     HeaderDetails,
     LiveStatusIndicator,
 } from '@jenkins-cd/design-language';
+import i18nTranslator from '../i18n/i18n';
+
+const translate = i18nTranslator('blueocean-web');
```
Comment (sophistifunk) [inline blueocean-core-js/src/js/components/ResultPageHeader.jsx]: Can we replace this with a prop? The containing component already has a `t` function and global consts complicate testing and reuse.

Comment (NicuPascu) [inline blueocean-core-js/src/js/components/ResultPageHeader.jsx]: yup, on it, thanks

**Comment by NicuPascu** [comment]
@i386 @michaelneale as requested
recording_2017-08-01_142749

**Comment by sophistifunk** [comment]
Looks good 🐝

**Comment by sophistifunk** [comment]
Now all we need is to ditch the close button icon for failed builds 😁

**Comment by michaelneale** [comment]
once green - get it in! nice one!

**Comment by kzantow** [comment]
@sophistifunk 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍

**Pull request merged**